Chummer/lang/en-us.xml

```diff
@@ -482,6 +482,10 @@
       <key>String_Delete</key>
       <text>Delete</text>
     </string>
+    <string>
+      <key>String_DeleteShort</key>
+      <text>X</text>
+    </string>
```

Comment:
No. Find a way to put an image into your button or control or whatever, then use the same image that's used in all other places for removing stuff (minus sign in a red circle). This is going to look extremely hacky and amateurish when used the way I think you're using it.